<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Equipe 3 - Digital Twin</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(32, 16, 110);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="1" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="title">
          <string>Options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>240</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QPushButton" name="btnSettings">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Settings</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnAquaCropSim">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>AquaCrop Sim</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnCrops">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Crops</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnSensors">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Sensors</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnStat">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Statistics</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QTabWidget" name="tabWidget">
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 85, 127);</string>
               </property>
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="tab">
                <attribute name="title">
                 <string>Summary</string>
                </attribute>
               </widget>
               <widget class="QWidget" name="tab_2">
                <attribute name="title">
                 <string>Constraints</string>
                </attribute>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QProgressBar" name="progressBar">
               <property name="value">
                <number>24</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QTabWidget" name="tabActuators">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 85, 127);</string>
            </property>
            <property name="tabPosition">
             <enum>QTabWidget::East</enum>
            </property>
            <property name="currentIndex">
             <number>4</number>
            </property>
            <property name="tabBarAutoHide">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="tab_setting">
             <attribute name="title">
              <string>Settings</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QFrame" name="frame_6">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QGridLayout" name="gridLayout_7">
                 <item row="0" column="0">
                  <widget class="QGroupBox" name="groupBox_3">
                   <property name="styleSheet">
                    <string notr="true">font:bold;color:white;</string>
                   </property>
                   <property name="title">
                    <string>GroupBox</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QGroupBox" name="groupBox_4">
                   <property name="styleSheet">
                    <string notr="true">font:bold;color:white;</string>
                   </property>
                   <property name="title">
                    <string>GroupBox</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QPushButton" name="btnApplySetting">
                   <property name="font">
                    <font>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: white;</string>
                   </property>
                   <property name="text">
                    <string>Apply</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_sim">
             <attribute name="title">
              <string>Simulation</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <widget class="QFrame" name="frame_7">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QGridLayout" name="gridLayout_9">
                 <item row="0" column="0">
                  <widget class="QGroupBox" name="groupBox_5">
                   <property name="styleSheet">
                    <string notr="true">font:bold;color:white;</string>
                   </property>
                   <property name="title">
                    <string>GroupBox</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QGroupBox" name="groupBox_6">
                   <property name="styleSheet">
                    <string notr="true">font:bold;color:white;</string>
                   </property>
                   <property name="title">
                    <string>GroupBox</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QGroupBox" name="groupBox_7">
                   <property name="styleSheet">
                    <string notr="true">font:bold;color:white;</string>
                   </property>
                   <property name="title">
                    <string>GroupBox</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_sensors">
             <attribute name="title">
              <string>Sensors</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="1">
               <widget class="QGroupBox" name="groupBox_8">
                <property name="styleSheet">
                 <string notr="true">color:white;font:bold;</string>
                </property>
                <property name="title">
                 <string>Sensors</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <layout class="QGridLayout" name="gridLayout_10">
                 <item row="0" column="0">
                  <widget class="QGroupBox" name="groupBox_9">
                   <property name="title">
                    <string>GroupBox</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QGroupBox" name="groupBox_16">
                   <property name="title">
                    <string>GroupBox</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QGroupBox" name="groupBox_10">
                   <property name="title">
                    <string>GroupBox</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QGroupBox" name="groupBox_17">
                   <property name="title">
                    <string>GroupBox</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QGroupBox" name="groupBox_11">
                   <property name="title">
                    <string>GroupBox</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="2">
                  <widget class="QGroupBox" name="groupBox_12">
                   <property name="title">
                    <string>GroupBox</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QGroupBox" name="groupBox_15">
                   <property name="title">
                    <string>GroupBox</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QGroupBox" name="groupBox_13">
                   <property name="title">
                    <string>GroupBox</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="2">
                  <widget class="QGroupBox" name="groupBox_14">
                   <property name="title">
                    <string>GroupBox</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_Actuator">
             <attribute name="title">
              <string>Actuators</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_11">
              <item row="0" column="0">
               <widget class="QGroupBox" name="groupBox_18">
                <property name="styleSheet">
                 <string notr="true">font:bold;color:white;</string>
                </property>
                <property name="title">
                 <string>Actuators</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <layout class="QGridLayout" name="gridLayout_12">
                 <item row="0" column="0">
                  <widget class="QGroupBox" name="groupBox_19">
                   <property name="title">
                    <string>GroupBox</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QGroupBox" name="groupBox_20">
                   <property name="title">
                    <string>GroupBox</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QGroupBox" name="groupBox_21">
                   <property name="title">
                    <string>GroupBox</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QGroupBox" name="groupBox_22">
                   <property name="title">
                    <string>GroupBox</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_stat">
             <attribute name="title">
              <string>Statistics</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QFrame" name="frame_8">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QGridLayout" name="gridLayout_13">
                 <item row="0" column="0">
                  <widget class="QGroupBox" name="gb_stat_1">
                   <property name="styleSheet">
                    <string notr="true">font:bold;color:white;</string>
                   </property>
                   <property name="title">
                    <string>Graphique 1</string>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_14">
                    <item row="0" column="0">
                     <widget class="PlotWidget" name="stat_graph_1" native="true"/>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QGroupBox" name="gb_stat_2">
                   <property name="styleSheet">
                    <string notr="true">font:bold;color:white;</string>
                   </property>
                   <property name="title">
                    <string>Graphique 2</string>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_15">
                    <item row="0" column="0">
                     <widget class="PlotWidget" name="stat_graph_2" native="true"/>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QGroupBox" name="gb_stat_3">
                   <property name="styleSheet">
                    <string notr="true">font:bold;color:white;</string>
                   </property>
                   <property name="title">
                    <string>Graphique 3</string>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_16">
                    <item row="0" column="0">
                     <widget class="PlotWidget" name="stat_graph_3" native="true"/>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 12pt &quot;MS Shell Dlg 2&quot;;
padding: 12pt;
color: white;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Digital Twin - Système de micro-irrigation</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>814</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
